ui/raidboss/data/03-hw/trial/sophia-ex.txt

```diff
@@ -1,157 +1,147 @@
-630	"Cintamani" #AoE
-637	"Arms of Wisdom" #Tank Buster
-643	"Thunder II"
-653	"Dischordant Damnation" #before 6 stack partner
-658	"Gnosis" #knockback
-670	"Arms of Wisdom"
-681	"Scale Tethers"
-685	"Dischordant Damnation" #before 6 stack partner
-692	"Tilt"
-696	"Shadows Spawn"
-700	"Cintamani" #AoE
-702	"Cintamani" #AoE
-704	"Cintamani" #AoE
-710	"Arms of Wisdom" #Tank Buster
-714	"Add Tethers"
-716	"Thunder"
-719	"Duplicate"
-720	"Add Tethers"
-722	"Aero III"
-725	"Duplicate"
-735	"Quasar (on arena)" #before 10 stack
-742	"Arms of Wisdom"
-753	"Dischordant Damnation" #before 6 stack partner
-756	"Execute"
-
-#Repeat loop of phase 5
-768	"Cintamani" #AoE
-770	"Cintamani" #AoE
-772	"Cintamani" #AoE
-779	"Arms of Wisdom" #Tank Buster
-781	"Scale Tethers"
-792	"Tilt"
-806	"Thunder III"
-815	"Arms of Wisdom"
-817	"Scale Tethers"
-829	"Tilt"
-831	"Onrush" #half arena dash + vulnerability
-844	"Cintamani" #AoE
-846	"Cintamani" #AoE
-848	"Cintamani" #AoE
-854	"Arms of Wisdom" #Tank Buster
-860	"Thunder II"
-871	"Dischordant Damnation" #before 6 stack partner
-876	"Gnosis" #knockback, stack SE
-888	"Arms of Wisdom"
-899	"Scale Tethers"
-903	"Dischordant Damnation"
-910	"Tilt"
+hideall "--Reset--"
+hideall "--sync--"
+
+0.0 "--Reset--" sync / 21:........:40000010:/ window 10000 jump 0
+
+0 "Start"
+1.8 "--sync--" sync /:Sophia:19C3:/ window 1.8,2
+12.0 "Thunder II" sync /:Sophia:19B0:/ window 12,5
+22.4 "Gnosis" sync /:Barbelo:19C2:/
+41.4 "Arms Of Wisdom?" sync /:Sophia:19C4:/ # May be skipped depending on DPS
+
+# This section will be reached either at 56.0, or at the time she is pushed to 90%
+56.0 "--Clones Appear--" sync / 03:........:Added new combatant Aion Teleos/ window 56,5
+63.6 "Thunder II/Thunder III" sync /:Sophia:19(AC|B0):/
+63.6 "Duplicate" sync /:Aion Teleos:19AB:/ window 30,2.5
+69.7 "Aero III" sync /:Sophia:19AE:/
+69.7 "Duplicate" sync /:Aion Teleos:19AB:/
+79.8 "Execute" sync /:Sophia:19AA:/
+#79.8 "Execute" sync /:Aion Teleos:19AF:/
+90.2 "Arms Of Wisdom" sync /:Sophia:19C4:/
+
+# Intermission.
+# Divine Spark can be pushed and thus is not synced.
+# Gnostic Rant and Infusion are purely HP-based pushes.
+110.6 "Cloudy Heavens" sync /:Sophia:19BE:/ window 110.6
+111.6 "--Untargetable--"
+120.9 "Horizontal Kenoma/Vertical Kenoma" sync /:The First Demiurge:19B[BC]:/
+121.9 "Gnostic Spear" sync /:The Third Demiurge:19B9:/
+125.9 "Divine Spark" # sync /:The Second Demiurge:19B6:/
+129.0 "Ring Of Pain" sync /:The Third Demiurge:19BA:/
+135.1 "Horizontal Kenoma/Vertical Kenoma" sync /:The First Demiurge:19B[BC]:/
+147.3 "Gnostic Spear?" sync /:The Third Demiurge:19B9:/
+149.2 "Horizontal Kenoma/Vertical Kenoma" sync /:The First Demiurge:19B[BC]:/
+191.6 "Zombification Enrage?"
+
+# There's an introductory block leading into a rotation.
+250.0 "The Scales Of Wisdom" sync /:Sophia:1981:/ window 250,5
+267.5 "The Scales Of Wisdom" sync /:Sophia:1AE1:/
+272.9 "--Targetable--"
+280.8 "Quasar (Meteor Snapshot)" sync /:Sophia:19(6E|A7):/
```

Comment:
What about `Quasar (Snapshot)` vs `Quasar (Meteor)` vs `Quasar (Tilt)` for brevity? Feel free to disagree.  :)

Reply:
You're right, that does always follow a `Quasar (Meteor)` line, so it wouldn't be ambiguous.